apps/picsa-apps/dashboard/src/app/modules/translations/pages/edit/translations-edit.component.ts

```diff
@@ -0,0 +1,69 @@
+import { CommonModule } from '@angular/common';
+import { Component } from '@angular/core';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { ActivatedRoute,Router } from '@angular/router';
+import { RouterModule } from '@angular/router';
+// eslint-disable-next-line @nx/enforce-module-boundaries
+import type { Database } from '@picsa/server-types';
+
+import { DashboardMaterialModule } from '../../../../material.module';
+// import { DashboardResourcesStorageLinkComponent } from '../../components/storage-link/storage-link.component';
+import { TranslationDashboardService } from '../../translations.service';
+
+type ITranslationEntry = Database['public']['Tables']['translations']['Row'];
+
+@Component({
+  selector: 'dashboard-translations-edit',
+  standalone: true,
+  imports: [CommonModule, DashboardMaterialModule, FormsModule, ReactiveFormsModule,RouterModule],
+  templateUrl: './translations-edit.component.html',
+  styleUrls: ['./translations-edit.component.scss'],
+})
+export class TranslationsEditComponent {
+  translationRow: ITranslationEntry;
+  dataLoadError: string;
+  editActionFeedbackMessage: string;
+  constructor(private service: TranslationDashboardService, private route: ActivatedRoute, private router: Router) {
+    this.service.ready();
+    this.route.params.subscribe((params) => {
```

Comment:
nit(non-blocking) - nice implementation to track route params. Just 2 minor issues:

1. Any time you create a subscription you are using browser memory. So when the subscription is no longer required you should remove it. This is commonly done by linking the subscription to ngOnInit and ngOnDestroy methods, e.g. 
https://blog.bitsrc.io/6-ways-to-unsubscribe-from-observables-in-angular-ab912819a78f

2. I've actually broken this implementation/page due to changes to how the `id` field is calculated. I'd say no point fixing right now as we will want to first decide what will be the best user flow for updating translations

Reply:
Thanks for this, will surely tidy up next time